ui/lib/apps/ContinuousProfiling/pages/ConProfSettingForm.tsx

```diff
@@ -21,12 +22,24 @@ import { useClientRequest } from '@lib/utils/useClientRequest'
 import { ErrorBar, InstanceSelect } from '@lib/components'
 import { useIsWriteable } from '@lib/utils/store'
 
+const ONE_DAY_SECONDS = 24 * 60 * 60
 const RETENTION_SECONDS = [
-  3 * 24 * 60 * 60,
-  5 * 24 * 60 * 60,
-  10 * 24 * 60 * 60,
+  3 * ONE_DAY_SECONDS,
+  5 * ONE_DAY_SECONDS,
+  10 * ONE_DAY_SECONDS,
 ]
```

Comment:
Considering that our translation key only support "day", how about we write `RETENTION_DAYS` here directly? Then you don't need to worry about the case that `second % DAYS != 0`.

Reply:
Because in the API, the `data_retention_seconds` filed value uses the `second` as the unit.

Reply:
How about:

```jsx
const RETENTION_DAYS = [3, 5, 10];

// ....

{RETENTION_DAYS.map((day) => (
  <Select.Option key={day} value={val * ONE_DAY_SECONDS}>
    {t('xxxxx', { d: day })}
  </Select.Option>
))}
```

Reply:
Looks good, let me update it in the next PR.